src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,114 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "BobBlink";
+    private static final String HUGE_LOGIN = "BobBlinkBobBlinkBobBlink";
+    private static final String NULL_VALUE = null;
+    private static final int VALID_AGE = 25;
+    private static final int LESS_AGE = -5;
+    private static final int HIGH_AGE = 100;
+    private static final String VALID_PASSWORD = "123456789";
+    private static final String NOT_VALID_PASSWORD = "123";
+    private static final User NULL_USER = null;
+    private static RegistrationService registrationService;
+    private static User user;
+    private static User user2;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+        user2 = new User();
+        user.setLogin(VALID_LOGIN);
+        user.setAge(VALID_AGE);
+        user.setPassword(VALID_PASSWORD);
```

Comment:
```suggestion

@BeforeEach
        user.setLogin(VALID_LOGIN);
        user.setAge(VALID_AGE);
        user.setPassword(VALID_PASSWORD);
```
Use beforeEach for this case. Than you can simplificate your tests by deleting rows like these